developer_manual/app_development/bootstrap.rst

```diff
@@ -76,7 +81,7 @@ The class **must** extend ``OCP\AppFramework\App`` and may optionally implement
             // ... boot logic goes here ...
 
             /** @var IManager $manager */
-            $manager = $context->getAppContainer()->query(IManager::class);
+            $manager = \OC::$server->getNotificationManager();
```

Comment:
That is the old way of fetching dependencies. The `query` one is correct.

Reply:
This is from https://github.com/nextcloud/notifications/blob/master/docs/notification-workflow.md#L14.
I guess the documents on Notifications repo should be updated.